services/cloudflareService.js

```diff
@@ -0,0 +1,23 @@
+const { fetch } = require("../utils/fetch");
+
+const CLOUDFLARE_ZONE_ID = config.get("cloudflare.CLOUDFLARE_ZONE_ID");
+const CLOUDFLARE_PURGE_CACHE_API = `https://api.cloudflare.com/client/v4/zones/${CLOUDFLARE_ZONE_ID}/purge_cache`;
+
+async function purgeCache(files) {
+  const response = await fetch(
+    CLOUDFLARE_PURGE_CACHE_API,
+    "POST",
+    null,
+    { files: files },
```

Comment:
isn't this for hosts/prefixes?

As per the documentation, there is a hosts parameter in the request
[Cloudflare API Documentation | Caching](https://developers.cloudflare.com/api/operations/zone-purge-files-by-cache-tags,-host,-or-prefix)

Reply:
We are using our fetch function for this.
https://github.com/Real-Dev-Squad/website-backend/blob/develop/utils/fetch.js
![image](https://user-images.githubusercontent.com/78433013/210127987-c9e1ffd7-cb04-4386-a1f6-bd071bb5632b.png)

Reply:
Does it mean files serve as `options` here?

Reply:
`files` serves as `data` or you can say it's `options` only but a specified option here